src/components/formio/radio.tsx

```diff
@@ -67,6 +78,22 @@ export const Radio: React.FC<RadioProps> = ({
       </div>
 
       {description && <Description text={description} />}
+
+      {values[name] && isClearable && (
```

Comment:
`values[name]` is too naive because dotted paths can be used which lead to nested lookups.

Use:

```tsx
const {getFieldProps} = useFormikContext();
const {value} = getFieldProps(name);
```
instead